repos/system_upgrade/el7toel8/actors/yumupgradability/tests/test_yumupgradability.py

```diff
@@ -0,0 +1,17 @@
+from subprocess import call
+
+from leapp.libraries.actor import yumupgradability
+
+
+def test_secure_yum_upgradability(monkeypatch):
+    """
+    Test Secure Yum upgradability library function
+
+    :return: Indicates Pass/Fail - whether tests succeeded
+    """
+
+    monkeypatch.setattr(yumupgradability, "CMDS", [['whoami'], ['ls']])
+    monkeypatch.setattr(yumupgradability, 'run_cmd', call)
+    expected_yellow_flag = False
+    actual_yellow_flag = yumupgradability.secure_yum_upgradability()
+    assert actual_yellow_flag == expected_yellow_flag
```

Comment:
could be written directly as `assert yumupgradability.secure_yum_upgradability() == False`

I would expect some tests where the function returns the flag

Reply:
You're right. I just wanted self-explanatory variables which refer to the return value of the tested function.